src/App.jsx

```diff
@@ -1,5 +1,26 @@
+import { Header } from './components/Header/Header';
+import { Courses } from './components/Courses/Courses';
+import { Navigate, Route, Routes } from 'react-router-dom';
+import { Login } from './components/Login/Login';
+import { Registration } from './components/Registration/Registration';
+import { CourseInfo } from './components/CourseInfo/CourseInfo';
+import { CreateCourse } from './components/CreateCourse/CreateCourse';
+
 function App() {
-	return <div>React</div>;
+	return (
+		<>
+			<Header />
+			<Routes>
+				<Route path='*' element={<Navigate to='/' />} />
+				<Route path='/' element={<Courses />} />
+				<Route path='/login' element={<Login />} />
+				<Route path='/registration' element={<Registration />} />
+				<Route path='/courses/:courseId' element={<CourseInfo />} />
+				<Route path='/courses' element={<Courses />} />
```

Comment:
/courses, /cources/{id} , /courses/add should not be available if the user is not logged in